docs/content/setup/linux.md

```diff
@@ -32,11 +32,11 @@ On Windows, AppImage can be run in WSL 2.
     ```bash
     chmod u+x InfectionMonkey-v1.13.0.AppImage
     ```
-1. Start Monkey Island by running the Infection Monkey AppImage package:
+2. Start Monkey Island by running the Infection Monkey AppImage package:
```

Comment:
Don't change these numbers.

You can write numbered lists in markdown that look like
```md
1. first
1. second
1. third
```

It will render like
```md
1. first
2. second
3. third
```

The benefit of this is that we can insert items into the list without needing to change any of the numbers.